### PR TITLE
Add HTTP Status Code errors

### PR DESCRIPTION
**Pull request by WilliamXieMSFT**
Add commonly seen HTTP Status Code errors and proposed workarounds during Snapshot Debugger attach.

(You can replace all of this text with your description.)

Before creating your pull request, please check your content against these quality criteria:

- Did you consider search engine optimization (SEO) when you chose the title in the metadata section and the H1 heading (i.e. the displayed title that starts with a single #)?
- For new articles, did you add it to the table of contents?
- Did you update the "ms.date" metadata for new or significantly updated articles?
- Are technical terms and concepts introduced and explained, and are acronyms spelled out on first mention?
- Should this page be linked to from other pages or Microsoft web sites?

For more information about creating content for docs.microsoft.com, see the contributor guide at https://docs.microsoft.com/contribute/.

### CODE REVIEW

**Comment by PRMerger14** [comment]
@WilliamXieMSFT : Thanks for your contribution! The author, @mikejo5000, has been notified to review your proposed change.

**Comment by WilliamXieMSFT** [comment]
@Mikejo5000 Please

**Comment by PRMerger6** [comment]
@WilliamXieMSFT : Thanks for your contribution! The author, @mikejo5000, has been notified to review your proposed change.

**Comment by PRMerger6** [comment]
@WilliamXieMSFT : Thanks for your contribution! The author, @mikejo5000, has been notified to review your proposed change.

**Comment by Mikejo5000** [comment]
@poppastring Hey Mark, how do these changes look?

**Comment by poppastring** [comment]
@Mikejo5000 This is excellent! Thanks

**Comment by Mikejo5000** [comment]
@WilliamXieMSFT Thanks for the doc updates!

**Comment by Mikejo5000** [comment]
#sign-off

**Comment by PRMerger17** [comment]
@rjagiewich : Thanks for your contribution! The author, @mikejo5000, has been notified to review your proposed change.

File: docs/debugger/debug-live-azure-apps-troubleshooting.md [hunk 1]
```diff
@@ -16,6 +16,75 @@ ms.workload:
...
+* Verify your site has not migrated to new instances. Snapshot Debugger uses the notion of ARRAffinity for routing requests to specific instances which can produce this error intermittently.
+* Contact us at the above alias if this error continues to persist.
+
+### (409) Conflict
+
+This error indicates the request conflicts with the current server state.
+
+We have a known issue when users attempt to attach Snapshot Debugger against an AppService that has enabled ApplicationInsights. ApplicationInsights sets the AppSettings with a different casing than Visual Studio, causing this issue.
+::: moniker range=">= vs-2019"
+We have resolved this in the latest version of VS 2019.
```
Comment (gewarren) [inline docs/debugger/debug-live-azure-apps-troubleshooting.md]: As an FYI, VS needs to be spelled out as Visual Studio. I'll fix this.

File: docs/debugger/debug-live-azure-apps-troubleshooting.md [hunk 2]
```diff
@@ -16,6 +16,75 @@ ms.workload:
...
+
+Take these steps:
+::: moniker range="< vs-2019"
+* Verify in the Azure portal if the AppSettings for SnapshotDebugger (SNAPSHOTDEBUGGER_EXTENSION_VERSION) and InstrumentationEngine (INSTRUMENTATIONENGINE_EXTENSION_VERSION) are uppercase. This will need to be manually updated and will force a site restart.
+::: moniker-end
+* Contact us at the above alias if this error continues to persist.
+
+### (500) Internal Server Error
+
+This error indicates the site is completely down or the server cannot handle the request. Snapshot Debugger only functions on running applications. [Application Insights Snapshot Debugger](https://docs.microsoft.com/en-us/azure/azure-monitor/app/snapshot-debugger) provides snapshotting on exceptions and may be the tool for your needs.
```
Comment (gewarren) [inline docs/debugger/debug-live-azure-apps-troubleshooting.md]: Here, the link should not contain a locale (en-us). I'll fix this.

**Comment by Mikejo5000** [comment]
@gewarren Thanks. I'll do an edit pass on changes once it's synced to the private repo.

**Comment by gewarren** [comment]
@WilliamXieMSFT Thanks for providing this valuable info. As an FYI, MS employees should use the private version of the repo (in this case, visualstudio-docs-pr) to make contributions. We have the ability to get spelling and grammar checks there (using Acrolinx) as well as previews of the built doc.

**Comment by WilliamXieMSFT** [comment]
@gewarren Thanks for the tips, I'll make sure to keep them in mind for future changes!

**Pull request merged**